toolchain/check/handle_call_expression.cpp

```diff
@@ -78,10 +78,10 @@ auto HandleCallExpression(Context& context, Parse::Node parse_node) -> bool {
                       context.params_or_args_stack().PeekCurrentBlockContents(),
                       return_storage_id, function_decl->parse_node,
                       callable.implicit_param_refs_id, callable.param_refs_id);
-  auto call_node_id = context.AddNode(
+  SemIR::InstId call_inst_id = context.AddInst(
```

Comment:
Preserving `auto` for now:
```suggestion
  auto call_inst_id = context.AddInst(
```